src/test/SeatsioSeatingChart.test.js

```diff
@@ -137,4 +137,11 @@ describe("SeatsioSeatingChart", () => {
             ));
         });
     });
+
+    it('it rerenders if props change', () => {
```

Comment:
You could try ```it('it rerenders if props change', () => {
        let chartComponent = mount(<SeatsioSeatingChart />);
        expect(chartComponent.find('#chart').length).toBe(2)
        
        chartComponent.setProps({ id: "anotherId" })

        expect(chartComponent.find('#chart').length).toBe(0)
        expect(chartComponent.find('#anotherId').length).toBe(2)
    });```

But i'm not entirely sure.. it feels like we should assert a change on the actual chart instead of the id of the container.

Reply:
You could change the `selectedObjects` property. After rerendering, you could assert that the passed in objects are now selected. Or resolve a promise when `onObjectSelected` is triggered.

Reply:
Sure, that could work. I'll give it a try. Just so I learn from this too, why is that better than the current implementation?

```
chartComponent.setProps({ onRenderStarted: () => resolve() });
```

It looks simple and elegant to me, but maybe it isn't so in the world of testing (or this could give us false positives?) @mroloux 

Thanks!

Reply:
Yes, good question. By only checking that onChartRendered is invoked, we don't check whether the configuration of the chart has actually changed. We might have only rerendered, without passing in the updated props to seats.io.

So checking that as well would make sense imho.

Reply:
OH I just realized `onRenderStarted` is not a chart method, but seatsio-react method. Well definitely wouldn't test against that. Alright I'll make the changes and let you all know.